lesson07/assignment/hello-world/src/components/HistoryPage.vue

```diff
@@ -0,0 +1,92 @@
+        </div>
+      </section>
+      <section>
+        <h2>Coffee Travels to Europe</h2>
+        <p>By the 17th century, through trade coffee finally made its way to Europe with many fearing this new dark beverage that had come from the East. Through this controversy, the Pope intervened and after finally tasting it, its popularity finally spread. Coffee began to replace many other drinks such as alcohol in the mornings, due to its ability to allow the drinker to be energized and alert throughout the day.</p>
+        <h2>Coffee In the US</h2>
+        <p>In the mid-1600s, coffee was brought to New York and although many coffee houses started to appear, tea was still superior among the British. After events of the Boston Tea Party, coffee was forever stamped in American culture.</p>
+      </section>
+      <section>
+        <aside>
+          <h2>Coffee Now</h2>
+          <p>Coffee nowadays can be found in many forms and methods. We'll share the evolution of coffee and how it brought us where we are today.</p>
+        </aside>
+        <div class="image-container">
+          <img src="images/ingredients-making-coffee-flat-lay.jpg" alt="ingredients for coffee" class="img-fluid">
+          <p>Coffee became a staple in American culture especially during the Industrial Revolution which is when the first wave of coffee can be seen. The last few decades have been a period of growth and evolution for the global coffee industry. </p>
+        </div>
+      </section>
+      <section>
+        <h2>Evolution of Coffee: The Three Waves</h2>
+        <h3>First Wave</h3>
+        <p>The first wave of coffee is seen as the lowest quality and is known as commodity coffee. The beans are artificially or naturally flavored, primarily pre-ground offerings, and the coffee is dark and bitter. <a href="https://www.drivencoffee.com/blog/coffee-waves-explained/" data-bs-toggle="tooltip" title="Click here to learn more about First Wave Coffee">Learn more</a></p>
+        <h3>Second Wave</h3>
+        <p>The second wave is largely focused on the experience and can be seen in brands like Starbucks and Caribou Coffee that revolutionized cafe culture in the United States in the late 1900s which introduced coffee lovers to a wider variety of coffee experiences. <a href="https://www.drivencoffee.com/blog/coffee-waves-explained/" data-bs-toggle="tooltip" title="Click here to learn more about Second Wave Coffee">Learn more</a></p>
+        <h3>Third Wave</h3>
+        <div class="image-container">
+          <img src="images/close-up-cappuccino-coffee-with-art-latte-wooden-table.jpg" alt="latte on table" class="img-fluid">
```

Comment:
This relative link is looking for an images folder in the current folder which is components.  Use ../images/close-up..

`<img src="../images/coffee-beans-top-view-background.jpg" alt="coffee" class="img-fluid">`

Reply:
This applies to any other images that aren't showing up and that are not dynamically coming from your data object.